src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -39,7 +40,13 @@ partial class WorkspaceViewModel
 
         internal ConnectorViewModel ActiveConnector
```

Comment:
You can rename it to `FirstActiveConnector` because we always return the first connector from the list.